packages/views/utilities/externalLinks.js

```diff
@@ -0,0 +1,32 @@
+import history from '@common/utilities/history';
+
+const sendRegex = /^\/(wallet|wallet\/send|main\/transactions\/send)$/;
+const sendRedirect = '/wallet?modal=send';
+
+const voteRegex = /^\/(main\/voting\/vote|delegates\/vote|vote)$/;
+const voteRedirect = '/wallet?modal=votingQueue';
+
+export default {
+  init: () => {
+    const { ipc } = window;
+
+    if (ipc) {
+      ipc.on('openUrl', (action, url) => {
+        const [protocol, rest] = url.split(':/');
```

Comment:
it may be better to use existing libraries for parsing URL like the below or any other do the job

https://www.npmjs.com/package/url-parse
https://github.com/ljharb/qs

Reply:
@soroushm While a library might be best way to handle urls, is it best to handle it here alone or to create a ticket to resolve such across the entire app?

Reply:
@soroushm Implemented using the URL API